alibi_detect/cd/mmd_online.py

```diff
@@ -150,3 +150,16 @@ def score(self, x_t: Union[np.ndarray, Any]) -> float:
         Squared MMD estimate between reference window and test window.
         """
         return self._detector.score(x_t)
+
+    def get_config(self) -> dict:  # Needed due to self.x_ref being a torch.Tensor when backend='pytorch'
```

Comment:
Need to override `get_config` in order to convert `self.x_ref` back to `np.ndarray` (it is a `torch.Tensor` in the pytorch version of `MMDDriftOnline`).

Reply:
Based on offline discussion with @mauicv, we've decided to leave this one for now since it is quite specific to a very small number of detectors.